task.py

```diff
@@ -342,6 +347,19 @@ def get_pod_ip(self) -> str:
         r = self.run_oc(f"get pod {self.pod_name} -o yaml", die_on_error=True)
         y = yaml.safe_load(r.out)
         return typing.cast(str, y["status"]["podIP"])
+    
+    def get_secondary_ip(self) -> str:
+        jsonpath = '{.metadata.annotations.k8s\.ovn\.org\/pod-networks}'
+        logger.info(f"get pod {self.pod_name} -o jsonpath='{jsonpath}'")        
+        r = self.run_oc(f"get pod {self.pod_name} -o jsonpath='{jsonpath}'")
+        if r.returncode != 0:
+            logger.info(r)
+            sys.exit(-1)
```

Comment:
drop the `if r.returncode != 0` block and pass `die_on_error=True` to `self.run_oc()`.